modules/platforms/dotnet/Apache.Ignite.Tests/Table/RecordViewPocoTests.cs

```diff
@@ -721,6 +721,34 @@ public async Task TestAllColumnsPocoNullable()
             Assert.AreEqual(poco.DateTime, res.DateTime);
         }
 
+        [Test]
+        public async Task TestEnumColumns()
+        {
+            // Normal values.
+            await Test(new PocoEnums.PocoIntEnum(1, PocoEnums.IntEnum.Foo));
+            await Test(new PocoEnums.PocoByteEnum(1, PocoEnums.ByteEnum.Foo));
+            await Test(new PocoEnums.PocoShortEnum(1, PocoEnums.ShortEnum.Foo));
+            await Test(new PocoEnums.PocoLongEnum(1, PocoEnums.LongEnum.Foo));
+
+            // Values that are not represented in the enum (it is just a number underneath).
```

Comment:
We can also test the default value, like
await Test(new PocoEnums.PocoIntEnum(1, default(PocoEnums.IntEnum)));

Reply:
Good point, added.